kpi/deployment_backends/kobocat_backend.py

```
@@ -230,13 +233,18 @@ def calculated_submission_count(self, user: 'auth.User', **kwargs) -> int:
 
     @property
     def current_month_submission_count(self):
+        try:
+            xform_id = self.xform_id
+        except InvalidXFormException:
+            return 0
```

Fail early instead of nesting conditionals :heart_eyes: